### DISCUSSION TITLE
embed and control

### DISCUSSION THREAD

**Discussion by quantrpeter** [coder/code-server#6820] [Q&A]
Hi
   I want to embed code-server in a php page and provide some buttons outside it to control the loading. When I cliuck on the button, it refresh vscode web and load a specific folder, and run some commands in terminal. Is it possible? thanks

Just like this one https://tanstack.com/table/v8/docs/framework/react/examples/bootstrap?fbclid=IwZXh0bgNhZW0CMTAAAR1wrxDtdPxtA_u3Fe2w9QAy9rqFoi9qOFSIBeXG4FdG7V5630aF1FHE3-E_aem_AWeigzgas6sm11DjLTKxnmp_AMFbxWbBnjkKss6jjBtiu92iPdEgJgSlVUqLSy5_eNbRsEgaZmnRDH4FqnoYuSwj

**Comment by code-asher** [comment]
You can embed code-server and load a specific folder, but there is no way to run commands from an parent frame as far as I know.
